protocol/app/app.go

```diff
@@ -1683,6 +1683,8 @@ func (app *App) PreBlocker(ctx sdk.Context, _ *abci.RequestFinalizeBlock) (*sdk.
 
 // BeginBlocker application updates every begin block
 func (app *App) BeginBlocker(ctx sdk.Context) (sdk.BeginBlock, error) {
+	ctx = ctx.WithExecMode(lib.ExecModeBeginBlock)
```

Comment:
wanna do 
```
	ctx = log.AddPersistentTagsToLogger(ctx,
		log.Handler, XXXXXX,
		log.BlockHeight, ctx.BlockHeight(),
	)
```
while you are at it?

Reply:
use the strings not the ints though

Reply:
you mean to move this from `x/clob`'s begin blocker to here?

Reply:
can just add some tags to the app-level app.go file

Reply:
going to punt on this for now so that I can merge. let's do this in a separate PR